ru_RU.json

```diff
@@ -0,0 +1,956 @@
+  "FORCE_WANTED_LVL": "Уровень розыска",
+  "FORCE_WANTED_LVL_INFO": "Держать определенный уровень розыска активным для себя",
+  "FREECAM": "Свободная камера",
+  "FREECAM_DESC": "Отсоединяет камеру от игрока и позволяет свободно перемещать ее.",
+  "FREEZE_LOOP": "Постоянная заморозка",
+  "FRONT": "Передний",
+  "FULLY_AUTOMATIC_SIGNAL": "Полностью автоматический сигнал",
+  "FUN_VEHICLE_SEAT": "Сиденье ",
+  "FUN_VEHICLE_SPEED": "Скорость({})",
+  "FUN_VEHICLE_TOP_SPEED": "Максимальная скорость({})",
+  "GAME_CACHE": "Кэш игры",
+  "GAME_CACHE_DONT_CARE": "Мне все равно, обновляйтесь в одиночной игре!",
+  "GAME_CACHE_GO_ONLINE": "Обновите кеш в онлайне.",
+  "GAME_CACHE_ON_INIT": "Обновить кеш при инициализации",
+  "GAME_CACHE_SINGLE_PLAYER_DESCRIPTION": "В настоящее время вы находитесь в одиночной игре, вы можете принудительно создать кеш в одиночной игре, но рискуете вылететь при переходе в многопользовательскую игру или загрузить онлайн и кэшировать.",
+  "GAME_CACHE_UPDATE": "YimMenu требует пересборки кеша игры. ",
+  "GAME_CACHE_UPDATE_CACHE": "Обновить кеш",
+  "GAME_CACHE_UPDATING": "Обновление кеша, подождите...",
+  "GAME_CACHE_WAITING_FOR_ONLINE": "Ожидание онлайн, чтобы начать обновление кеша...",
+  "GAME_CACHE_WAITING_FOR_SINGLE_PLAYER": "Ожидание загрузки одиночной игры...",
+  "GENERAL": "Общее",
+  "GET_ALL_WEAPONS": "Получить все оружие",
+  "GIVE_AMMO": "Дать патроны",
+  "GIVE_AMMO_DESC": "Дает игроку боеприпасы",
+  "GIVE_ARMOR": "Дать броню",
+  "GIVE_ARMOR_DESC": "Дает игроку броню",
+  "GIVE_HEALTH": "Дать здоровье",
+  "GIVE_HEALTH_DESC": "Дает игроку немного здоровья",
+  "GIVE_WEAPONS": "Дать оружие",
+  "GIVE_WEAPONS_ALL_DESC": "Дает всем оружие. ",
+  "GIVE_WEAPONS_DESC": "Дает все оружие игроку",
+  "GOD_MODE": "Режим бога",
+  "GODMODE": "Режим бога",
+  "GODMODE_DESC": "Предотвращает получение любого вида урона",
+  "GOLF": "Гольф",
+  "GTA_BANNER": "Баннер GTA",
+  "GUI_TAB_BLACKHOLE": "Черная дыра",
+  "GUI_TAB_CONTEXT_MENU": "Контекстное меню",
+  "GUI_TAB_CREATOR": "Создатель",
+  "GUI_TAB_CUSTOM_TELEPORT": "Пользовательский телепорт",
+  "GUI_TAB_DEBUG": "Отлаживать",
+  "GUI_TAB_ESP": "ESP",
+  "GUI_TAB_ESP_SETTINGS": "ESP",
+  "GUI_TAB_GTA_CACHE_SETTINGS": "Кэш ГТА",
+  "GUI_TAB_GUI": "графический интерфейс",
+  "GUI_TAB_GUI_SETTINGS": "графический интерфейс",
+  "GUI_TAB_HANDLING": "Умение обращаться",
+  "GUI_TAB_HANDLING_CURRENT_PROFILE": "Текущий профиль",
+  "GUI_TAB_HANDLING_SAVED_PROFILE": "Сохраненные профили",
+  "GUI_TAB_HOTKEY_SETTINGS": "Горячие клавиши",
+  "GUI_TAB_IPL": "IPL",
+  "GUI_TAB_LSC": "ЛС кастомс",
+  "GUI_TAB_LUA_SCRIPTS": "Lua-скрипты",
+  "GUI_TAB_MISSIONS": "Миссии",
+  "GUI_TAB_MOBILE": "Мобильный",
+  "GUI_TAB_MODEL_SWAPPER": "Замена модели",
+  "GUI_TAB_NEARBY": "Рядом",
+  "GUI_TAB_NETWORK": "Сеть",
+  "GUI_TAB_ORBITAL_DRONE": "Орбитальный дрон",
+  "GUI_TAB_OUTFIT_EDITOR": "Редактор костюмов",
+  "GUI_TAB_OUTFIT_SLOTS": "Слоты для экипировки",
```

Comment:
GUI_TAB_OUTFIT_EDITOR and GUI_TAB_OUTFIT_SLOTS are inconsistent. Slots says it's for equipment not for outfits. Beep boop, Mr. Machine Translation.